src/layouts/SiteLaunchPad/components/SiteLaunchInfoGathering.tsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { Box, Button, Icon, Text, RadioGroup } from "@chakra-ui/react"
+import { Input, Radio } from "@opengovsg/design-system-react"
+import { useState } from "react"
+import { useForm } from "react-hook-form"
+import { BiRightArrowAlt } from "react-icons/bi"
+
+import { SITE_LAUNCH_PAGES } from "types/siteLaunch"
+
+import { SiteLaunchPadBody } from "./SiteLaunchPadBody"
+import { SiteLaunchPadTitle } from "./SiteLaunchPadTitle"
+
+export const SiteLaunchInfoGatheringTitle = (): JSX.Element => {
```

Comment:
nit: file names for react components are ususally nouns. can perhaps name as SiteLaunchInfoCollector or similar

Reply:
fd4a79af74d7ef722eabf4d61885b2ba300ed96f